src/color.scss

```diff
@@ -0,0 +1,100 @@
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
+
+/* stylelint-disable no-descending-specificity, declaration-empty-line-before */
+
+.popover {
+  border-color: var(--ds-auro-dropdown-popover-border-color);
+  background-color: var(--ds-auro-dropdown-popover-background-color);
+  box-shadow: -2px 0 5px 2px rgba(0 0 0 / .08), 0 2px 5px 1px rgba(0 0 0 / .08);
+  color: var(--ds-auro-dropdown-popover-text-color);
+}
+
+.label {
+  color: var(--ds-auro-dropdown-label-text-color);
+}
+
+.trigger {
+  border-color: var(--ds-auro-dropdown-trigger-border-color);
+  background-color: var(--ds-auro-dropdown-trigger-background-color);
+  color: var(--ds-auro-dropdown-trigger-text-color);
+
+  &:focus-within,
+  &:active {
+    --ds-auro-dropdown-trigger-border-color: var(--ds-color-border-active-default, #{$ds-color-border-active-default});
+  }
+
+  &:focus-within {
+    &:not(:active) {
+      --ds-auro-dropdown-trigger-border-color: var(--ds-color-ui-default-default, #{$ds-color-ui-default-default});
+      box-shadow: inset 0 0 0 1px var(--ds-auro-dropdown-trigger-border-color);
+    }
+  }
+}
+
+.helpText {
+  color: var(--ds-auro-dropdown-help-text-color);
+}
+
+:host([disabled]) {
+  --ds-auro-dropdown-trigger-text-color: var(--ds-color-text-ui-disabled-default, #{$ds-color-text-ui-disabled-default});
+  --ds-auro-dropdown-label-text-color: var(--ds-color-text-ui-disabled-default, #{$ds-color-text-ui-disabled-default});
+}
+
+:host([common]),
+:host([bordered]) {
+  .trigger {
+    --ds-auro-dropdown-trigger-border-color: var(--ds-color-border-secondary-default, #{$ds-color-border-secondary-default});
```

Comment:
I don't think this token value set here on 51 needs to be in any of the selectors of 51 and can be moved up a level.

Reply:
Can you explain this more in depth? Im not sure how I would move it up a level

Reply:
Doesnt need to be within `.trigger` selector

Reply:
While your comment makes sense, we do actually need to `.trigger` selector here. We don't need it to change the value of the token on line 51, but we need it to change the value of the token on lines 53-55. In this case, it also wouldn't make sense to move the token on line 51 out of the `.trigger` selector.